packages/circuits/circuits/qr_verify.circom

```diff
@@ -2,14 +2,17 @@ pragma circom 2.1.6;
 
 include "./rsa.circom";
 include "./sha.circom";
+include "./filter.circom";
 
 
 template QR_Verify(n, k, len) {
     signal input padded_message[len]; // private
     signal input message_len; // private 
     signal input signature[k]; //private
     signal input modulus[k]; //public
+    signal input selector[16];
 
+    signal output reveal_data[len];
```

Comment:
Do we need to output this? 
As for now, it is just to compute the nullifier.

Reply:
This only for extract basic information, sorry because made you confused